### ISSUE TITLE
How can I add accuracy during training and validation?

### ISSUE THREAD

**Issue by Crista23** [hunkim/word-rnn-tensorflow#57]
I would like to see if the model improves during training or is overfitting. To this end, I would like on the one hand to add accuracy as a metric besides loss, and a validation set to see the validation loss and accuracy. I am new to Tensorflow, is there any easy way to do it? Thanks!

**Comment by abubakar-ucr** [comment]
Did someone post a solution to this? or @Crista23, did you find something yourself.
